docs/components/time.md

```diff
@@ -0,0 +1,182 @@
+---
+id: time
+title: Network Time System
+description: An explanation of the new network time system that replaces the old NetworkTime. This system will form the foundation for building other systems such as snapshoots, commands and predictions
+---
+
+<!---## Deprecations / Removals
+
+### NetworkManager Settings
+
+`Receive Tickrate`, `Network Tick Interval Sec`, `Max Receive Events Per Tick Rate`, `Event Tickrate` settings in NetworkManager are getting removed.
+
+
+
+- `Network Tick Interval Sec` will now be given via the new `Tickrate` field.
+- `Receive Tickrate` is being removed. There is no replacement for this. MLAPI will now poll the receive queues at the beginning of every tick.
+- `Event Tickrate` will now be given via the new `Tickrate` field.
+- `Max Receive Events Per Tick Rate` will be removed. There will be no functionality to restrict the amount of events in MLAPI anymore as we move from an event to a snapshot based system.
+
+The default value for `Tickrate` will be `60` instead of `64`. A tickrate of 60 is more consistent with industry standard tickrates. Note that this rate is different than the default tickrate for FixedUpdate in Unity which is 50.
+
+### NetworkTime
+
+`NetworkTime` will be removed from MLAPI. As explained later in the RFC there are some flaws in the concept of a single `NetworkTime` so we will be replacing it with other concepts instead. For a quick fix of existing code instead of:
+```csharp
+var time = NetworkManager.Singleton.NetworkTime;
+```
+use
+```csharp
+var time = NetworkManager.Singleton.PredictedTime.Time;
+```
+
+## New Features
+
+-->
```

Comment:
Why is this hidden? Being saved for something specific?

Reply:
its being hidden as singleton is being removed